src/main/java/nl/novi/techItEasy/controllers/TelevisionController.java

```diff
@@ -55,4 +56,9 @@ public ResponseEntity<Television> removeTelevisionBasedOnId(@PathVariable Long i
         return new ResponseEntity<>(null, HttpStatus.NO_CONTENT);
     }
 
+    @PutMapping("{id}/remote")
```

Comment:
Bij de andere PUT-mapping (ik kan daar geen comment toevoegen) heb je de @Valid annotatie bij de @Pathvariable staan, maar dat is geen DTO en heeft dus geen validatie regels om te valideren. @Valid is alleen voor DTO's met validatieregels.